tests/compatibility/forward_test.go

```diff
@@ -0,0 +1,243 @@
+// Copyright 2023 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compatibility_test
```

Comment:
Why add a new package?

Reply:
use client 6.6 rather than master

Reply:
What if we wanna test another version? Do we need to add more and more modules?

Reply:
I am not sure. But now tests for api server forward tso request is invalid with new client.

Reply:
If we are going to merge it, I think we need to add comments to explain the situation.

Reply:
add more comments, ptal

Reply:
where do we specify using client 6.6?

Reply:
> where do we specify using client 6.6?

`github.com/tikv/pd/client v0.0.0-20230213111801-7d0389306a8b`